src/scenes/UserGalleryPage/UserGalleryCollection.tsx

```diff
@@ -262,10 +262,6 @@ const StyledCollectorsNote = styled(BaseM)<{ showMore: boolean }>`
           }
 
           -webkit-line-clamp: 2;
-
```

Comment:
@Robinnnnn we only want to remove this if we're tablet size and greater. logic is a bit hard to follow but basically the idea is if we're doing line clamping, we don't want the padding, and on > tablet, we don't do the line clamp

Reply:
thanks @tbezman, fixed! tested on both desktop + mobile